akka-projection-core/src/main/scala/akka/projection/internal/SourceProviderAdapter.scala

```diff
@@ -30,9 +30,9 @@ import akka.stream.scaladsl.Source
     // it _should_ not be used for the blocking operation of getting offsets themselves
     val ec = akka.dispatch.ExecutionContexts.parasitic
     val offsetAdapter = new Supplier[CompletionStage[Optional[Offset]]] {
-      override def get(): CompletionStage[Optional[Offset]] = offset().map(_.toJava)(ec).asJava
+      override def get(): CompletionStage[Optional[Offset]] = offset().map(_.asJava)(ec).toJava
```

Comment:
This is astonishing! 

I never realised that Option moved from `toJava` to `asJava` while Future from `asJava` to `toJava`. Let's wait for 2.14, we may need to revert it again.